### PR TITLE
nheko: 0.8.1 -> 0.8.2

### PR DESCRIPTION
**Pull request by pstn**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/nheko/default.nix [hunk 1]
```diff
@@ -30,13 +29,13 @@
 
 mkDerivation rec {
   pname = "nheko";
-  version = "0.8.1";
+  version = "0.8.2-RC";
```
Comment (Ekleog) [inline pkgs/applications/networking/instant-messengers/nheko/default.nix:32]: If this is still an RC, now is probably a bit early to bump the nixpkgs version?

**Review by Ekleog — CHANGES_REQUESTED** [review]

File: pkgs/applications/networking/instant-messengers/nheko/default.nix (hunk 1 above)
Comment (pstn) [reply pkgs/applications/networking/instant-messengers/nheko/default.nix:32]: Indeed, this is why this is just a draft.

**Comment by Ekleog** [comment]
@SuperSandro2000 I see you merged. Out of curiosity, what checks did you perform to validate the lmdbxx owner change? My searching the internet appears to say that as of today, lmdbxx in the mind of the public would still refer to the drycpp one, even though the hoytech one has started to appear on search engines.

**Comment by SuperSandro2000** [comment]
It is required by the neko update.
https://github.com/Nheko-Reborn/nheko/commit/59b6712e2868b9f7be65d32a50ce373ce54d404b#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR104

**Comment by Ekleog** [comment]
Hmm… I mean I understand why the change is necessary in order to bump nheko; but I feel uneasy about having our default lmdbxx be a version that may not be the one people would by default think it is.

I'm thinking it may be better to keep the old lmdbxx, and have an lmdbxx-cpp17 or equivalent for the new version; and once the world will have switched to the new lmdbxx make lmdbxx-cpp17 be the default. What do you think?

**Comment by SuperSandro2000** [comment]
nheko is the only consumer of lmdbxx. I don't think we need an older version of library that is not used anywhere.

**Comment by fgaz** [comment]
And this fork was already adopted by gentoo.

By the way, anyone knows why I did not get the maintainer notification for the lmdbxx change?

File: pkgs/applications/networking/instant-messengers/nheko/default.nix [hunk 2]
```diff
@@ -47,7 +46,6 @@ mkDerivation rec {
 
   buildInputs = [
     nlohmann_json
-    tweeny
```
Comment (fgaz) [inline pkgs/applications/networking/instant-messengers/nheko/default.nix:50]: The changelog says there is also a new dependency: xcb

File: pkgs/development/libraries/lmdbxx/default.nix [hunk 3]
```diff
@@ -4,13 +4,13 @@
 
 stdenv.mkDerivation rec {
   pname = "lmdbxx";
-  version = "0.9.14.0";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
-    owner = "drycpp";
+    owner = "hoytech";
```
Comment (fgaz) [inline pkgs/development/libraries/lmdbxx/default.nix:10]: The homepage url should be modified too (I can't comment on that line)

File: pkgs/applications/networking/instant-messengers/nheko/default.nix (hunk 2 above)
Comment (fgaz) [reply pkgs/applications/networking/instant-messengers/nheko/default.nix:50]: /cc @ekleog @fpletz

but maybe it was already included tranitively

**Comment by Ekleog** [comment]
I hadn't noticed that the fork was already adopted by gentoo for lmdb++-1.0, thank you for the precision! Keeping things as is totally makes sense then :)

**Pull request merged**